ale_linters/cpp/cquery.vim

```diff
@@ -0,0 +1,34 @@
+" Author: Ben Falconer <ben@falconers.me.uk>
+" Description: A language server for C++
+
+call ale#Set('cpp_cquery_executable', 'cquery')
+call ale#Set('cpp_cquery_cache_directory', expand('~/.cache/cquery'))
+
+function! ale_linters#cpp#cquery#GetProjectRoot(buffer) abort
+    let l:project_root = ale#path#FindNearestFile(a:buffer, 'compile_commands.json')
```

Comment:
`.cquery` as the filename is another, seemingly simpler, option here. It works if I simply replace the filename in `cquery.vim`, but I haven't been able to figure out how to make it use either.